Source/WebKit/UIProcess/Cocoa/AuxiliaryProcessProxyCocoa.mm

```diff
@@ -75,10 +75,10 @@
 }
 
 #if USE(EXTENSIONKIT)
-RetainPtr<_SEExtensionProcess> AuxiliaryProcessProxy::extensionProcess() const
+ExtensionProcess AuxiliaryProcessProxy::extensionProcess() const
 {
     if (!m_processLauncher)
-        return nullptr;
+        return ExtensionProcess();
```

Comment:
FYI, I believe this calls the std::variant's default constructor, which creates a value with the variant's first alternative initialized. As a result, I believe it returns a std::variant containing a `RetainPtr<BEWebContentProcess>` (because that's the first alternative) which is `nullptr`.

As a result, I doubt your checks that use `variant_npos` (or `valueless_by_exception()`) do anything here?